specification/consumption/resource-manager/readme.md

```diff
@@ -336,3 +347,16 @@ regenerate-manager: true
 generate-interface: true
 ```
 
+### Tag: package-2019-01 and java
```

Comment:
please take a look at the readme file for each of the other language SDKs located in the same folder as readme.md and update those too. Java setting are included in this file, but the others have been extracted out.

Reply:
Please check that all languages are generating SDKs for the PR, and that the generated code is resulting as expected (you should see links to PRs for each language posted as messages in this PR).